oak_functions/examples/weather_lookup/Dockerfile

```
@@ -1,6 +1,8 @@
-# Allow deploying the latest version for development.
-# hadolint ignore=DL3007
-FROM gcr.io/oak-ci/oak-functions:latest
+# Use an argument to specify container identifier.
+# Default to using the latest version for development.
+ARG container_identifier=:latest
```

How about

```suggestion
ARG base_image_tag=latest
```

(consistent with https://docs.docker.com/engine/reference/commandline/tag/)

or just make the entire image name an arg.

---

Using just a tag will not allow us to specify a specific hash. I will change to using the arg to cover the entire name.